### DISCUSSION TITLE
Date Picker - Can be entered from the keyboard?

### DISCUSSION THREAD

**Discussion by arkdev1** [wireui/wireui#793] [Q&A]
Good morning, I'm trying to modify the DateTime (I only use Date, without time) to leave the field free for keyboard entry. Ideas? I haven't found any settings for this.

**Comment by insulae** [comment]
> Good morning, I'm trying to modify the DateTime (I only use Date, without time) to leave the field free for keyboard entry. Ideas? I haven't found any settings for this.

I have the same requirement, were you able to solve it? I have users who don't select the date, they type it in and it's a problem :-/

**Comment by insulae** [comment]
Apparently this was a bug and it was fixed: https://github.com/wireui/wireui/issues/985

but I updated to version 2.5.1 and the problem persists (unless I'm misunderstanding how to use it).
